src/com/epam/javacourse/homework2/HomeWork.java

```diff
@@ -0,0 +1,35 @@
+package com.epam.javacourse.homework2;
+
+import java.util.Arrays;
+
+public class HomeWork {
+
+    public static void main(String[] args) {
+        // Fibonacci
```

Comment:
It's ok to use space between `//` and the word as well as not to use it. But the approach is required to be common for all the comments throughout your project.
Pay attention to lines 12, 13, etc.

Reply:
Fixed.